src/Lachain.Core/CLI/ConsoleCommands.cs

```diff
@@ -221,20 +222,21 @@ public string SignTransaction(string[] arguments)
             var value = Money.Parse(arguments[2]);
             var from = _keyPair.PublicKey.GetAddress();
             var tx = _transactionBuilder.TransferTransaction(from, to, value);
-            var signedTx = _transactionSigner.Sign(tx, _keyPair);
+            var signedTx = _transactionSigner.Sign(tx, _keyPair, HardforkHeights.IsHardfork_8Active(_blockManager.GetHeight()));
```

Comment:
I think it should be _blockManager.GetHeight() + 1, if it goes to the next block

Reply:
resolved